scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+
+ROOTFS_PATH="$(pwd)/bin/targets/x86/64/openwrt-x86-64-generic-rootfs.tar.gz"
+NETWORK_PREFIX="${NETWORK_PREFIX:-192.168.1}"
+NETWORK_NAME="openwrt-lan-$NETWORK_PREFIX"
+IMAGE_NAME="openwrt-rootfs:$NETWORK_PREFIX"
+
+[ -f "$ROOTFS_PATH" ] || echo "Couldn't find rootfs at $ROOTFS_PATH"
+
+cat <<EOT > "$(dirname $ROOTFS_PATH)/Dockerfile"
+FROM scratch
+ADD $(basename $ROOTFS_PATH) /
+RUN sed 's/pi_ip="192.168.1.1/pi_ip="$NETWORK_PREFIX.1"/'
+RUN sed 's/pi_broadcast="192.168.1.255/pi_broadcast="$NETWORK_PREFIX.255"/'
+RUN echo "console::askfirst:/usr/libexec/login.sh" >> /etc/inittab
+EXPOSE 22 80 443
+USER root
+CMD ["/sbin/init"]
+EOT
+
+echo "[*] Build: $ROOTFS_PATH"
+
+if [ -z "$(docker network ls | grep $NETWORK_NAME)" ]; then
+	docker network create \
+	  --driver=bridge \
+	  --subnet="$NETWORK_PREFIX.0/24" \
+	  --ip-range="$NETWORK_PREFIX.0/24" \
+	  --gateway="$NETWORK_PREFIX.2" \
+	  "$NETWORK_NAME"
+	echo "[*] Created $NETWORK_NAME network "
+fi
+
+docker build -t "$IMAGE_NAME" "$(dirname $ROOTFS_PATH)"
```

Comment:
No removal/cleanup of the image?

Reply:
No, do you think this is really required? Individual images are automatically removed by using --rm

Reply:
That image has to be stored somewhere, so if you don't remove it, then they'll pile up somewhere.